tools/build_pak_files.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import glob
+import os
+import subprocess
+import sys
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--unreal_binaries_dir", required=True)
+    parser.add_argument("--unreal_project_dir", required=True)
+    parser.add_argument("--scene_id", required=True)
```

Comment:
scene_id should be optional

Reply:
Hmmm, currently our perforce project contains only 1 scene in content folder. As we add more scenes, the same content folder will grow and contain all the scenes. So, building a separate pak for each scene from this content folder would not be possible as far as I know (i.e., in this scenario, I'm not sure we can single out a scene for this pak generation process).

If building a separate pak for each scene is desirable (in the long run, with 300 or more scenes, I think this would be the better option), then we will have to change the current Perforce project directory structure, and update this script to generate pak file for each scene separately.

If building a single pak file containing all provided scenes is desirable, then, we do not need `scene_id` at all. We will name the final pak as something like `SpearSim-Content-v...-Win64.pak`.

What do you think?

Reply:
I believe it is desirable to build a pak file for each scene, and we should have a Perforce folder for each scene. This will also be desirable when you want to edit a specific scene in the Unreal Editor. You should be able to create a symlink in `SpearSim/Content` that refers to a specific scene to facilitate editing that specific scene.